### ISSUE TITLE
Packaged application issues error "“Uncaught Exception: Error: ENOENT: no such file or directory, open 'settings.json'", but not the original application

### ISSUE THREAD

**Issue by ZenoFox** [electron-userland/electron-packager#776]
Just created a package of an open source application using the electron-packager, but now I'm getting this error when opening the generated package:

    Uncaught Exception: Error: ENOENT: no such file or directory, open 'settings.json
    

The main.js error shows up when running this:

    ipcMain.on('setupUi', function(event) {
      fs.readFile('settings.json', (err, data) => {
        settings = JSON.parse(data);
        mainWin.webContents.send('setupUi', JSON.parse(data));
      });
      fs.readFile( 'billing.json', (err, data) => {
        profiles = JSON.parse(data);
        for (p in profiles) {
          switch(profiles[p].country) {
            case 'US':
              profilesUS[p] = profiles[p];
              break;
            case 'UK':
              profilesUK[p] = profiles[p];
          }
        }
      });
    })

The error does not show up when running the original application. It only shows up in the version created by the electron packager. I've tried to manually add *settings.json* and *billing.json* to the folder of the application that was produced by the electron packager, the error still shows up.

Do I somehow have to mention settings.json and billing.json in the package.json file? How do I do this? Or how can I make sure the packaged applications finds those too?

Does anybody have an idea how I could fix this?

**Comment by malept** [comment]
This is a common error when packaging apps, please see the FAQ: https://github.com/electron-userland/electron-packager/blob/master/docs/faq.md#why-isnt-the-relative-path-in-my-app-code-working

**Comment by ZenoFox** [comment]
Okay, so I've now edited the code and it looks like this:
```
const path = require('path');
var settings = path.resolve(__dirname, 'settings.json');
var billing = path.resolve(__dirname, 'billing.json');
```

I still get the error.

I've also tried leaving out the "__dirname" and some other combinations.

The directory of the app looks like this:
https://imgur.com/a/8R0VP

Can't use "const" instead of "var", because that results in another error, also I want my app to be able to change the .json files.

I'd really appreciate if you help.
Sorry, it's probably just a very simple problem, I'm new to node.js and all that kind of stuff.
